### PR TITLE
Also include fixtures at users' locations

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?237323

### CODE REVIEW

**Comment by esoergel** [comment]
@czue @orangejenny

**Comment by orangejenny** [comment]
don't really have context for why this works, but it looks reasonable

**Comment by esoergel** [comment]
The user should receive fixtures assigned to their location or any ancestor location.  If a fixture is assigned to, say Suffolk, a Boston user should also see it.  That currently works, but right now `loc_ids` doesn't include the user's location, so a user at Suffolk wouldn't see a fixture assigned to Suffolk.

**Comment by orangejenny** [comment]
got it, thanks for explaining :)

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/fixtures/tests.py [hunk 1]
```diff
@@ -171,6 +172,105 @@ def test_get_indexed_items(self):
...
+            item_attributes=[],
+        )
+        data_type.save()
+
+        def make_data_item(location_name, cost):
+            """Make a fixture data item and assign it to location_name"""
+            data_item = FixtureDataItem(
+                domain=cls.domain,
+                data_type_id=data_type.get_id,
+                fields= {
```
Comment (dimagimon) [inline corehq/apps/fixtures/tests.py]: E251 unexpected spaces around keyword / parameter equals

File: corehq/apps/fixtures/tests.py [hunk 2]
```diff
@@ -171,6 +172,105 @@ def test_get_indexed_items(self):
...
+                            field_value=location_name,
+                            properties={},
+                        )]
+                    ),
+                },
+                item_attributes={},
+            )
+            data_item.save()
+
+            fixture_ownership = FixtureOwnership(
```
Comment (dimagimon) [inline corehq/apps/fixtures/tests.py]: F841 local variable 'fixture_ownership' is assigned to but never used

**Comment by esoergel** [comment]
@orangejenny Just added two more commits to future-proof this a bit.

**Comment by orangejenny** [comment]
new commits look good, hurrah for tests

**Comment by czue** [comment]
nice

**Pull request merged**